schemas/plugins/gauge/panel.cue

```diff
@@ -1,11 +1,24 @@
 package gauge
 
+import (
+	"github.com/perses/perses/schemas/plugins/common"
+)
+
 #panel: {
-  displayed_name: string
-  kind: "GaugeChart"
-  chart: {
-    expr: string
-  }
+	kind: "GaugeChart"
+	display: {
+		name: string
+	}
+	options: {
+		query:       #query
+		calculation: common.#calculation
+		unit?:       common.#unit
+		thresholds?: common.#threasholds
+	}
+}
+
+#query: {
+
```

Comment:
this is where we have to be able to inject new definition. Here for example we should be able to inject the prometheus query definition.

I'm wondering how we will do that